Tests/SPTPersistentCacheTests.m

```diff
@@ -1397,6 +1397,7 @@ - (void)testCurrentDataTimeInterval
     SPTPersistentCache *cache = [self createCacheWithTimeCallback:nil expirationTime:0];
     
     NSTimeInterval firstTimeInterval = [cache currentDateTimeInterval];
+    [NSThread sleepForTimeInterval:0.1]; // sleep a beat
```

Comment:
couldn't get this test to pass locally w/o a little sleep here. maybe my machine is just too fast 💪